creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1194,35 +1192,43 @@ class SnapbackSM {
    * @notice this will issue sync to healthy secondary and update replica set away from unhealthy secondary
    */
   async aggregateReconfigAndPotentialSyncOps(nodeUsers, unhealthyPeers) {
+    // Parallelize calling this._aggregateOpsWithQueriedSpIds on chunks of 500 nodeUsers at a time
+    const BATCH_SIZE = 500
```

Comment:
Followup on why this is 500 @vicky-g I think it's reasonably low since there aren't a lot of requests in the function it calls. The non-IO logic is just a few if-elses, and the IO is:
1. Redis (going to change in a future PR anyway): 1 call -> 500 calls per batch (Redis can take 10000 concurrent connections by default so a cap of 500 is plenty conservative)
2. Health check (probably going to be removed soon): for each user, max of 1 call to the user's 2 secondaries -> max of 1000 calls per batch spread out over 10+ nodes -> max of 100 calls to a node per batch

Reply:
thank you for this! I think number 1 sounds good, number 2 may be different if the peer set slice is only a subset of the nodes in the network. so, for example, 1000 calls spread over 5 nodes -> 200 calls. I think this is ok for now -- lets just monitor this in case it doesnt work out. good stuff

Reply:
ah good point! I'll keep an eye out for anything that looks related to rejected connections